_ont_xgs/ont-fs-XGS-ONU-25-20NI-cli.md

```diff
@@ -0,0 +1,222 @@
+---
+title: CLI command tree (XGS-ONU-25-20NI)
```

Comment:
please add the link in the main page

and please se a detal for reduce the size of command like https://hack-gpon.org/ont-huawei-ma5671a-rooted/

Reply:
ok,
This is left to @alvgalrus